### PR TITLE
Implementing possible implementation for #4

### PR DESCRIPTION
**Pull request by jrock2004**
I tested this fix on Mac Big Sur and tested on PopOS 21.10

### CODE REVIEW

File: getnf [hunk 1]
```diff
@@ -26,6 +27,12 @@ while getopts :hk option; do
     esac
 done
 
+# For Macs, need to set a few different things
+if [[ "$os" == 'Darwin' ]]; then
+    distDir="$HOME/Library/Fonts"
+    [ -e "$downDir" ] && echo "Fonts Directory exists, good" || mkdir -p "$downDir"
```
Comment (ronniedroid) [inline getnf]: this is redundant, as curl can create a directory with the `--create-dirs` flag

File: getnf [hunk 2]
```diff
@@ -48,6 +55,28 @@ for i in $listFonts; do
...
+									awk -F ":" 'FNR == 1 {print $2}' | \
+									awk '{print $1}' | \
+								        sed 's/"//g') &&
+						fc-list | grep -i $installedFontName | \
+						    awk -F "/" '{print $7}' | \
+						    sed 's/style\=//' |  \
+						    awk -F ":" 'BEGIN {print "FONT NAME" " | " "FILE NAME" " | " "STYLE"} {print $2 " | " $1 " | " $3}' | \
+						#set this variable to true so that the font cache get's updated
+            aFontInstalled=True)
+        else
```
Comment (ronniedroid) [inline getnf:79]: all of this is redundant as it is doing the same thing as in the else block, except the curl part, which I recommend not to do with a `cd` and just let curl output the file to the output directory which is set as the downDir

**Review by ronniedroid — CHANGES_REQUESTED** [review]
please take the suggested into consideration

File: getnf (hunk 1 above)
Comment (jrock2004) [reply getnf]: I’ll try again but it was not working for me on mac

File: getnf (hunk 2 above)
Comment (jrock2004) [reply getnf:79]: Output dir is not supported on mac

File: getnf (hunk 1 above)
Comment (jrock2004) [reply getnf]: Updated

File: getnf (hunk 2 above)
Comment (jrock2004) [reply getnf]: Updated

**Comment by ronniedroid** [comment]
Thank you a lot and sorry for the late merge.

**Pull request merged**